packages/gql/src/balancer-internal-manager/internalBalances.ts

```diff
@@ -9,7 +9,6 @@ export const internalBalance = gql`
           name
           symbol
           address
-          decimals
```

Comment:
not sure if I understood why you removed the decimals here

Reply:
nvm! you don't need the decimals here, right? this is the query you use to display user's internal balances and `balance` is already downscaled so you don't have to parse it.